unicodethingy.py

```diff
@@ -14,10 +14,10 @@
 
 print(counter.most_common(150))
 
-Ls = { chr(i) + ' (' + str(i) + ') ' + unicodedata.name(chr(i),'')
-        for i in range(0xFFFF)
-        if unicodedata.name(chr(i),'').split().__contains__('L') }
+arrows = { chr(i) + ' (' + str(i) + ') ' + unicodedata.name(chr(i),'')
```

Comment:
you really want to use `unichr()` here and in the various other lines I can't comment on (because they're not part of the PR) rather than `chr()`

Reply:
Also, as with many languages, using String.format() or a formatted string (look it up), depending on context, is usually considered preferable to use of the `+` operator for string concatenation.

Reply:
unichr is only for python2, which only ruminants and disreputable varmints use.

Reply:
Thanks, apparently OSX still needs to have python 2 around for Reasons(tm), I should have been typing 
`$ python3`
rather than 
`$ python`
to check out bits of code interactively.